test/orchestration_helpers.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const MONGODB_URI = process.env.MONGODB_URI;
```

Comment:
what is this file used for?

Reply:
It's currently not used for anything. It is a leftover relic from when I was trying to do full integration with mongo orchestration. I can remove it from this commit.